config/src/main/java/no/sysco/middleware/kafka/clients/interceptor/config/Utils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package no.sysco.middleware.kafka.clients.interceptor.config;
+
+import java.util.Collections;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.errors.TopicExistsException;
+
+/**
+ *
+ * @author 100tsa

```

Comment:
Could we remove this? Authors are on commits already. Maybe changing to `@author SYSCO Middleware team` would be better?

Reply:
fixed